apps/webapp/app/env.server.ts

```diff
@@ -27,7 +27,10 @@ const EnvironmentSchema = z.object({
   AUTH_GITHUB_CLIENT_SECRET: z.string().optional(),
   FROM_EMAIL: z.string().optional(),
   REPLY_TO_EMAIL: z.string().optional(),
-  RESEND_API_KEY: z.string().optional(),
+  SMTP_HOST: z.string().optional(), 
+  SMTP_PORT: z.number().optional(), 
```

Comment:
@Chigala Environment variables always come through as strings, so you need to do this:

```suggestion
SMTP_PORT: z.coerce.number().optional(), 
```

Reply:
I'm going to just make this change now so I can merge

Reply:
Thanks